SUPPORT.md

```diff
@@ -1,13 +1,28 @@
 # Support
 
-## How to file issues and get help  
+## Microsoft Support Policy
 
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion.
 
-For help and questions about using this project, please submit a Github issue with corresponding [Issue Labels found here](https://github.com/Azure/Enterprise-Scale/labels). Responses will be posted within 24 Hours. 
+Following list of issues are considered within realm when contacting Microsoft support:
 
-## Microsoft Support Policy  
+- Portal deployment of 4 Reference implementation - WingTip, Contoso and AdventureWorks, TreyResearch
+- Underlying RP issues when deploying template (e.g. MGs, Policies LA, vWAN, vNet) for any deployment failure
+- Subscription Creation via portal experience
+- UI elements (e.g., ArmApiController) used in ARM Template
+- RP Issues e.g. template validation, Checkaccess API etc.
+- Customization of Portal Template/Policy and UI Definition by customer/partners
 
-Support for this **PROJECT or PRODUCT** is limited to the resources listed above.
+Any issues that deemed outside of the above list by Microsoft support and/or requires bugfix in the Template or Code in the repo, Microsoft support will redirect user to file the issue on GitHub.
+
+Project maintainers and community aim to get issues resolved in timely fashion as per community support policy of this repo.
```

Comment:
Issue may take more than 3 days to resolve. For GitHub repos that are supported, our rule is "72 hours to get a meaningful response from the product team".